src/views/preremplissage.vue

```diff
@@ -0,0 +1,297 @@
+<script setup lang="ts">
+import LoadingModal from "@/components/loading-modal.vue"
+import InputDate from "@/components/input-date.vue"
+import InputDepcom from "@/components/input-depcom.vue"
+import { ref, computed } from "vue"
+import { useStore } from "@/stores/index.js"
+import { Answer } from "@lib/types/store.d.js"
+import axios from "axios"
+
+const store = useStore()
+
+const family_name = ref<string | undefined>()
+const given_names = ref<string | undefined>()
+const birthdate = ref<string | undefined>()
+const postcode = ref<string | undefined>()
+const gender = ref<string | undefined>()
+const email = ref<string | undefined>()
+const phone = ref<string | undefined>()
+const cityName = ref<string | undefined>()
+const updating = ref(false)
+const formError = ref(false)
+const prefillSuccess = ref(false)
+
+const handleNomCommuneUpdate = (nomCommune: string) => {
+  cityName.value = nomCommune
+}
+const handleCodePostalUpdate = (codePostal: string) => {
+  postcode.value = codePostal
+}
+
+const formData = computed(() => {
+  return {
+    family_name: family_name.value,
+    given_names: given_names.value,
+    birthdate: birthdate.value,
+    gender: gender.value,
+    birthplace_insee_code: postcode.value,
+    birthcountry_insee_code: "99100",
+    email: email.value || "",
+    phone: phone.value || "",
+  }
+})
+
+const formDataValidation = computed(() => {
+  if (
+    !formData.value.family_name ||
+    !formData.value.given_names ||
+    !formData.value.birthdate ||
+    !formData.value.gender ||
+    !formData.value.birthplace_insee_code ||
+    !formData.value.birthcountry_insee_code
+  ) {
+    return false
+  }
+  return true
+})
+
+const storeAnswer = (id: string, fieldName: string, value: any) => {
+  const answer: Answer = {
+    id,
+    entityName: "individu",
+    fieldName,
+    value,
+  } as Answer
+  store.answer(answer)
+}
+
+const storePivotDataAnswer = (data: { id: string; token: string }) => {
+  if (data.id && data.token) {
+    storeAnswer("demandeur", "pivot-data", data)
+  } else {
+    console.error("Missing response data API (id, token)", data)
+  }
+}
+
+const storeBirthdateAnswer = () => {
+  storeAnswer("demandeur", "date_naissance", birthdate.value)
+}
+
+const submitPrefillData = async () => {
+  try {
+    formError.value = false
+    prefillSuccess.value = false
+
+    if (!formDataValidation.value) {
+      formError.value = true
+      throw new Error("Missing required fields")
+    }
+
+    const pivotApiUrl: string | undefined = process.env.VITE_PIVOT_URL
+    if (!pivotApiUrl) {
+      throw new Error("Missing pivot api url")
+    }
+
+    updating.value = true
+    const response = await axios.post(
+      pivotApiUrl + "users",
```

Comment:
Je le remet après utilisation de axios 

```suggestion
      pivotApiUrl + "/users",
```

c'est plus conventionnel pour un endpoint je trouve (`\` à enlever de l'environemment du coup)

Reply:
C'est corrigé